### PR TITLE
Pretty up the login form.

### PR DESCRIPTION
**Pull request by chuckharmston**
r? @ngokevin 

screen shot 2015-09-29 at 12 37 40 pm

### CODE REVIEW

File: src/media/js/site/containers/login.js [hunk 1]
```diff
@@ -4,9 +4,10 @@ import {bindActionCreators} from 'redux';
 
 import {fxaLoginBegin, login} from '../actions/login';
 import {LoginButton} from '../components/login';
+import {Page} from '../components/page';
 
 
-export class Login extends React.Component {
+export class LoginForm extends React.Component {
```
Comment (ngokevin) [inline src/media/js/site/containers/login.js]: not really a fan. now half the codebase calls it Login, and the other half calls it LoginForm. and Login is implied to always be a form.

File: src/media/js/site/containers/__tests__/login.test.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import {LoginForm} from '../login';
+
+
+const props = {
```
Comment (ngokevin) [inline src/media/js/site/containers/__tests__/login.test.js]: we're passing some unused/junk props here

File: src/media/js/site/containers/login.js (hunk 1 above)
Comment (chuckharmston) [inline src/media/js/site/containers/login.js]: I did that because "login" is an action that's taken, and a container should be a noun. I can change it back.

File: src/media/js/site/containers/__tests__/login.test.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  authUrl: '',
+  fxaLoginBegin: () => {},
+  login: () => {},
+  user: {},
+  signTOS: () => {},
+  siteConfig: {},
+};
+
+describe('LoginForm', () => {
+  jsdom();
```
Comment (ngokevin) [inline src/media/js/site/containers/__tests__/login.test.js]: add line

File: src/media/js/site/containers/__tests__/login.test.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  it('renders', () => {
+    const component = ReactDOMHelper.render(
+      <StubRouterProvider Component={LoginForm}
```
Comment (ngokevin) [inline src/media/js/site/containers/__tests__/login.test.js]: this can fit

File: src/media/js/site/containers/__tests__/landing.test.js [hunk 5]
```diff
@@ -1,26 +1,21 @@
...
-    it('renders', () => {
-      const component = ReactDOMHelper.render(
-        <StubRouterProvider Component={Landing}
-                            {...props}/>
-      );
-      assert.equal(ReactDOMHelper.queryClassAll(component, 'landing').length, 1);
-    });
+  it('renders', () => {
+    const component = ReactDOMHelper.render(
+      <StubRouterProvider Component={Landing}
```
Comment (ngokevin) [inline src/media/js/site/containers/__tests__/landing.test.js]: this can fit in 80

File: src/media/js/site/containers/__tests__/landing.test.js [hunk 6]
```diff
@@ -1,26 +1,21 @@
 import {Landing} from '../landing';
 
 
+const props = {
```
Comment (ngokevin) [inline src/media/js/site/containers/__tests__/landing.test.js]: we're not using all these props either

File: src/media/css/containers/login.styl [hunk 7]
```diff
@@ -0,0 +1,20 @@
+@import '../lib'
+
+
+.login-form
+  background $color--white
+  border-radius 5px
+  max-width 400px
```
Comment (ngokevin) [inline src/media/css/containers/login.styl]: maybe 320px so we're sure it fits on mobile? could also add a var to lib/layout.styl

**Comment by ngokevin** [comment]
r+wc

File: src/media/js/site/containers/login.js (hunk 1 above)
Comment (ngokevin) [inline src/media/js/site/containers/login.js]: yeah could you? Components and actions don't collide in namespace, it's login vs Login, and we have lots of Components + Actions + Reducers that match each other's names

File: src/media/css/containers/login.styl (hunk 7 above)
Comment (chuckharmston) [inline src/media/css/containers/login.styl]: It's `max-width`, we're good:

screen shot 2015-09-29 at 1 39 15 pm

File: src/media/js/site/containers/__tests__/landing.test.js (hunk 6 above)
Comment (chuckharmston) [inline src/media/js/site/containers/__tests__/landing.test.js]: This was actually supposed to be in a separate commit entirely.

Comment (ngokevin) [inline src/media/js/site/containers/__tests__/landing.test.js]: it still stands. the Landing component does not use some of these props

Comment (chuckharmston) [inline src/media/js/site/containers/__tests__/landing.test.js]: Yep. Will be fixed in another commit.

File: src/media/js/site/containers/__tests__/login.test.js (hunk 3 above)
Comment (chuckharmston) [inline src/media/js/site/containers/__tests__/login.test.js]: There is a line below it?

File: src/media/js/site/containers/landing.js [hunk 8]
```diff
@@ -43,7 +45,7 @@ export class Landing extends React.Component {
               Submit a Firefox OS Add-On &rsaquo;
             </ReverseLink>
           </section>
-          {!this.props.user.token &&
+          {isLoggedOut &&
```
Comment (chuckharmston) [inline src/media/js/site/containers/landing.js]: This removes reliance on `this.props.user`.

Comment (ngokevin) [inline src/media/js/site/containers/landing.js]: well it's just an intermediate var to this.props.user

Comment (chuckharmston) [inline src/media/js/site/containers/landing.js]: `this.props.user` isn't required, so when it wasn't populated, a check for `this.props.user.token` would fail. Minor bug, test helped me find this :+1:
